builtin/merge-ours.c

```diff
@@ -11,12 +11,24 @@
 #include "git-compat-util.h"
 #include "builtin.h"
 #include "diff.h"
+#include "parse-options.h"
+
+static const char * const merge_ours_usage[] = {
+	N_("git merge_ours [arg]..."),
```

Comment:
First, please note that Git's commands use kebab-case instead of snake_case, i.e a dash instead of an underscore.

I don't think that the merge strategies receive any free-form arguments. Unfortunately, the documentation does not shed light into this: https://git-scm.com/docs/git-merge#_merge_strategies

While it _is_ possible to pass in options to merge strategies via the `-X` option, there does not seem to be any that is heeded by `git merge-ours`. So we can skip the `[options]` part that other synopses have.

As to the concrete parameters passed to merge strategies, not even the code that calls the merge strategies is helpful, either: https://github.com/git/git/blob/v2.23.0/merge.c#L17-L45

So let's look at a concrete merge strategy. The easiest might be `git-merge-octopus`, and indeed, this is helpful: https://github.com/git/git/blob/v2.23.0/git-merge-octopus.sh#L13. It says

> ```
> # The first parameters up to -- are merge bases; the rest are heads.
> ```

So maybe you want to use the synopsis `git merge-ours [<base>...] -- <head> <merge-head>...`?

Reply:
Thank you for your feedback @dscho! I'll use that test instead of the much longer
$ prove -j$(nproc) --shuffle t[0-9]*.sh

I'll make the suggested edits and read all those documents you have listed for insight :)